src/components/select/select.scss

```diff
@@ -68,7 +68,6 @@
         display: inline-flex;
         align-items: center;
 
-        cursor: pointer;
```

Comment:
There doesn't seem to be a matching mixin here? Not saying you did anything wrong, just wondering if there's something I'm missing, or if this was just not needed anymore? 🙂

Reply:
Good catch! I actually checked, it's further up or down in the code in the same file ;)

Reply:
Ah, yeah, it's probably this one:

<img width="517" alt="image" src="https://user-images.githubusercontent.com/154725/229448171-0e061a4e-17e2-4ea0-a734-e2c9e48afac4.png">